### ISSUE TITLE
Reproduce the baseline results

### ISSUE THREAD

**Issue by Srinivasa-Divakar-Bhat** [cdb342/Codes-for-IJCAI-2022-Paper-Zero-Shot-Logit-Adjustment#10]
Thank you for the amazing work and also for making the codes public. 

I would like to know how the results of the original f-CLSWGAN paper and that in your paper differs heavily. 
Is it primarily because of the prototype learner?
Also is it possible to reproduce these results using your code?

Thanks in advance.

**Comment by cdb342** [comment]
> Thank you for the amazing work and also for making the codes public.
> 
> I would like to know how the results of the original f-CLSWGAN paper and that in your paper differs heavily. Is it primarily because of the prototype learner? Also is it possible to reproduce these results using your code?
> 
> Thanks in advance.

Thank you for your kind words about our work.

In our paper, the results for f-CLSWGAN were generated using the original source code. The difference in performance on the CUB dataset is due to the use of different semantic vectors. While the original f-CLSWGAN paper used attributes provided by the CUB dataset, we reported results using 1024-dimensional character-based CNN-RNN features. For the SUN dataset, our results are close to the original paper. For the AWA dataset, the original paper reported results on AWA1, whereas we reported results on AWA2.
